preview.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<head>
+<link type="text/css" rel="stylesheet" href="css/preview_style.css"/>
+<script type="text/javascript" src="js/vendor/jquery-1.11.2.min.js"></script>
+<script type="text/javascript">
+function preview()
+{
+ $("#preview_div").html($("#text").val());
+}
+</script>
+</head>
+<body>
+<div id="wrapper">
+
+<textarea id="text"onkeyup="preview();" placeholder="Enter Text Or Code"></textarea>
+
+<div id="preview_div"></div>
```

Comment:
<div class="d-flex p-2">I'm a flexbox container!</div> this may solve the issue?
`<div class="d-flex p-2">I'm a flexbox container!</div>`